### ISSUE TITLE
Code of Merit is too long

### ISSUE THREAD

**Issue by dpyro** [rosarior/Code-of-Merit#19]
Less is more. The shorter the Code of Merit, the easier it is to understand and adopt. We will can spend more time debating less words in order to fashion a more precise code.

I propose reducing the CoM by merging similar bulletpoints into better abstractions.

**Comment by robe2** [comment]
Do you have some examples for shortening it? It's unclear to me what pieces can be collapsed together.  For what it's worth I've seen Cocs that are too short be dismissed by people as "gasp, being too short".

**Comment by odilitime** [comment]
Read PR, looks good but generally having finely grained enumeration is generally good for discussion and dissection.

**Comment by bittner** [comment]
A good example of a [short CoC](https://medium.com/@jmaynard/a-contribution-policy-for-open-source-that-works-bfc4600c9d83) is:

> We will accept contributions of good code that we can use from anyone.

(Credit to @brodock who mentioned this in https://github.com/rosarior/Code-of-Merit/issues/23#issuecomment-202155125. :+1:)

"Code" needs to be interpreted in a appropriate way for things like "link collections", e.g. the [awesome lists](https://github.com/sindresorhus/awesome), then we're all set. But clearly, the Code of Merit explains more that just what is said above. Also, what are the leveling rules according to "[good and] better code" is judged and identified. This needs some explanation to make discussions easier that come up inevitably, sooner or later.

The 14 bullets of the CoM are indeed many. Some of the scariness of the large count could be alleviated by highlighting the **essential statements** of the paragraphs.

TL;DR
-----
- **Some explanation** is needed (responsibilities, authorities, expected behavior).
- If this is all clear already, a **one-line CoC** as mentioned above **may suffice**.
- Highlight **essential statements** to allow for better readability (and potential, effective shortening).

**Comment by Djhg2000** [comment]
**Initial disclaimer; IANAL (and nor would I wish to be one).**

I started to cut it down to fit inside an 80x24 terminal and this is the initial idea:

    Code of Merit
    =============
    
    The managing members have final say in every decision of the project.
    Don't expect others to do your work or help you with your work forever.
    All members have the same opportunities within the project.
    Seniority must be earned.
    Technical advantage is the primary evaluation metric.
    Topics outside of the project will be ignored.
    Non technical conflicts will be discussed in a separate space.
    Individual characteristics are irrelevant and will not be taken into account.
    Discuss or debate the idea, not the person.
    It is the responsibility of the originator to provide requested context.
    This Code of Merit cannot take precedence over governing law.
    This Code of Merit only governs the technical procedures of the project.
    Participation on the project equates to agreement of this Code of Merit.
    No objectives beyond the stated objectives of this project are relevant.

This is when I realized there's a lot of redundancy in what's left. With the redundant parts combined into one line, it becomes something like this:

    Code of Merit
    =============
    
    The managing members have final say in every decision of the project.
    Don't expect others to do your work or help you with your work forever.
    Contributions are evaluated on technical merits.
    Seniority must be earned.
    Topics outside the scope of the project will be ignored.
    It is the responsibility of the originator to provide requested context.
    Discuss or debate the idea, not the person.
    This Code of Merit does not take precedence over governing law.
    This Code of Merit only governs the technical procedures of the project.
    Participation on the project equates to agreement of this Code of Merit.

But wait, there's more. Some non-redundant parts can still be combined if you share Torvalds' concept of respect:

    Code of Merit
    =============
    
    The managing members have final say in every decision of the project.
    Don't expect others to do your work or help you with your work forever.
    Contributions are evaluated on technical merits.
    Respect must be earned. Discuss or debate the idea, not the person.
    Topics outside the scope of the project will be ignored.
    It is the responsibility of the originator to provide requested context.
    This Code of Merit does not take precedence over governing law.
    This Code of Merit only governs the technical procedures of the project.
    Participation on the project equates to agreement of this Code of Merit.

At a grand total of 72x12, including the header and formatting, this heavily reduced version easily fits within the previously mentioned 80x24 terminal. Half a terminal page should be short enough for everyone but the laziest of developers.

I'm not saying we can't keep the full detailed version, in fact we would probably need it for clarifications and definitions. This could also be implemented like so:

    Code of Merit (reduced version, full text and definitions in CoM.txt)
    =============
    
    The managing members have final say in every decision of the project.
    Don't expect others to do your work or help you with your work forever.
    Contributions are evaluated on technical merits.
    Respect must be earned. Discuss or debate the idea, not the person.
    Topics outside the scope of the project will be ignored.
    It is the responsibility of the originator to provide requested context.
    This Code of Merit does not take precedence over governing law.
    This Code of Merit only governs the technical procedures of the project.
    Participation on the project equates to agreement of this Code of Merit.

I'm sure I've missed something, and utterly demolished all legal trickery, but I stand by my 80x24 terminal as a definition of what can be reasonably expected to be read in detail by everyone it applies to.
